### ISSUE TITLE
Build errors on Mac

### ISSUE THREAD

**Issue by sylvinus** [butterproject/butter#15]
In the `nwjs:src` step:
```
Create release folder in /Users/sylvinus/w/butter/butter/build/Butter/osx64
Fatal error: ENOENT, no such file or directory './src/app/images/butter.icns'
```

Fixed by renaming `src/app/images/popcorntime.icns` to `src/app/images/butter.icns`

Then after doing `grunt start`:
```
Running "start" task

Running "exec:mac" (exec) task
>> /bin/sh: build/cache/mac/0.12.3/nwjs.app/Contents/MacOS/nwjs: No such file or directory
```

`open build/Butter/osx64/Butter.app` works.

**Comment by jpgilchrist** [comment]
Hey @sylvinus did  you see any errors like the following:

```
Fatal error: Error: UNKNOWN, open '/Users/jpeg/Development/github/jpgilchrist/butter/build/Butter/osx64/Butter.app/Contents/Resources/app.nw/node_modules/opensubtitles-api/node_modules/lodash/internal/arrayReduce.js'
```

It's been happening to me on various lodash/internal files. It's never the same file up repeated executions of `grunt build`

**Comment by sylvinus** [comment]
nothing like that. the fixes I indicated made it work

**Comment by Mohitsharma44** [comment]
I tried it but I get following: 
```
Running "start" task

Running "exec:mac" (exec) task
>> Exited with code: 127.
>> /bin/sh: build/cache/mac/0.12.3/nwjs.app/Contents/MacOS/nwjs: No such file or directory
Warning: Task "exec:mac" failed. Use --force to continue.

Aborted due to warnings.
```

Trying to open it gives me: 
```
LSOpenURLsWithRole() failed with error -10665 for the file /Users/mohitsharma44/devel/personal/butter/build/Butter/osx64/Butter.app.
```

And opening it using launcher says: 
```
You can’t open the application “Butter.app” because PowerPC applications are no longer supported.
```

**Comment by xaiki** [comment]
fixed in fe0b8de thanks !

**Comment by jpgilchrist** [comment]
@xaiki 

```
Fatal error: Error: UNKNOWN, open '/Users/jpeg/Development/github/jpgilchrist/butter/build/Butter/osx64/Butter.app/Contents/Resources/app.nw/node_modules/nedb/node_modules/binary-search-tree/index.js'
```
